cocos/physics-2d/box2d-wasm/platform/physics-ray-cast-callback.ts

```diff
@@ -44,9 +44,9 @@ export class PhysicsRayCastCallback {// extends B2.RayCastCallback {
         PhysicsRayCastCallback._fractions.length = 0;
     }
 
-    static ReportFixture (fixture: number, point: B2.Vec2, normal: B2.Vec2, fraction: number): any {
+    static ReportFixture (fixture: number, point: B2.Vec2, normal: B2.Vec2, fraction: number): number {
         if ((B2.FixtureGetFilterData(fixture).categoryBits & PhysicsRayCastCallback._mask) === 0) {
-            return 0;
+            return -1;
```

Comment:
What does `-1` mean?

Reply:
Can refer to https://github.com/cocos/box2d_wasm/blob/wasm/include/box2d/b2_world_callbacks.h#L154 for more information.

```
-1 to filter, 0 to terminate, fraction to clip the ray for closest hit, 1 to continue
```